ibis/bigquery/compiler.py

```diff
@@ -394,6 +394,8 @@ def _formatter(translator, expr):
     ops.Capitalize,
     ops.DateDiff,
     ops.TimestampDiff,
+    ops.ExtractEpochSeconds,
```

Comment:
If you do want to implement this, it's the [`UNIX_SECONDS` function](https://cloud.google.com/bigquery/docs/reference/standard-sql/timestamp_functions#unix_seconds) in BigQuery